src/electron/main.js

```diff
@@ -1,10 +1,12 @@
 import { app, BrowserWindow, Menu, nativeTheme, protocol } from 'electron'
+import { fileURLToPath, URL } from 'url'
```

Comment:
I suggest to prefix NodeJS core modules with `node:`, to not confuse with packages from `node_modules`. Same for `path`, `fs`, etc.

```suggestion
import { fileURLToPath, URL } from 'node:url'
```